build.ps1

```diff
@@ -305,9 +309,9 @@ $dockerTagBase = "local"
 $dockerTagDMS = "$($dockerTagBase)/edfi-data-management-service"
 
 function DockerBuild {
-    Push-Location src/
-    &docker build -t $dockerTagDMS -f Dockerfile .
-    Pop-Location
+    # Push-Location src/
+    # &docker build -t $dockerTagDMS -f Dockerfile .
+    # Pop-Location
```

Comment:
Oh, this might explain my E2E failures. I had commented this out for faster debugging of the execution process.